rel-eng/tito.props

```diff
@@ -859,6 +859,7 @@ whitelist = ansiblerole-foreman_scap_client
 disttag = .el7
 scl = tfm
 whitelist = nodejs-babel-plugin-module-resolver
+  foreman-discovery-image-service
```

Comment:
technically this should be a line earlier, or the other node entry should be a line later, but I don't mind if you don't change this.

Reply:
We have scripts that automatically sort it when something is added. Not fixing this will create a diff on the next PR that modifies it. We really should have some lint check on it, but the current effort is focused on removing tito.props altogether so we're limited to manual reviews.